README.md

```diff
@@ -6,6 +6,25 @@
 
 > This is a preview version of the updated React Native Tools extension. It's not recommended to use the preview version and the stable one at the same time.
 
+## React Native Tools Preview migration guide
+
+In the Preview version of the extension we changed debug types in debug configurations. The table below shows the difference between debug configuration of the stable and the preview version.
+
+|RNT stable|RNT preview|
+|---|---|
+|```"type": "reactnative"```<br>![Android debug config stable](images/debug-config-android-stable.png)|```"type": "reactnative-preview"```<br>![Android debug config preview](images/debug-config-android-preview.png)|
```

Comment:
I think we can use ` instead of ``` here. Or are there any rendering issues with tables and HTML?

Reply:
I think we could also cut this table and just use a diff block to show users what they need to change, e.g.:

## Web debugging:
```diff
{
    "name": "Debug Android",
    "cwd": "${workspaceFolder}",
-    "type": "reactnative",
+    "type": "reactnative-preview",
    "request": "launch",
    "platform": "android"
}
```
## Direct debugging:
```diff
{
    "name": "Debug Android",
    "cwd": "${workspaceFolder}",
-    "type": "reactnativedirect",
+    "type": "reactnativedirect-preview",
    "request": "launch",
    "platform": "android"
}
```